iron-resizable-behavior.html

```diff
@@ -9,24 +9,24 @@
 -->
 
 <link rel="import" href="../polymer/polymer.html">
+<!--
+`IronResizableBehavior` is a behavior that can be used in Polymer elements to
+coordinate the flow of resize events between "resizers" (elements that control the
+size or hidden state of their children) and "resizables" (elements that need to be
+notified when they are resized or un-hidden by their parents in order to take
+action on their new measurements).
+Elements that perform measurement should add the `IronResizableBehavior` behavior to
+their element definition and listen for the `iron-resize` event on themselves.
+This event will be fired when they become showing after having been hidden,
+when they are resized explicitly by another resizable, or when the window has been
+resized.
+Note, the `iron-resize` event is non-bubbling.
+@group Polymer Behaviors
+@element iron-resizable-behavior
+@homepage github.io
```

Comment:
i've been removing homepage. should i be adding it back? 😓(this isn't actionable in this PR, just saying)